manifests/init.pp

```diff
@@ -43,14 +43,22 @@
 # Copyright 2017 Your name here, unless otherwise noted.
 #
 
+
 class ptpd(
 
   String $ptpinterface = 'eth0',
   Boolean $ptp_master = true,
+  $path = '/bin/:/sbin/:/usr/bin/:/usr/sbin/',
```

Comment:
There was a better solution, your acceptance tests passed but your rspec-puppet unit tests did not, your code was correct originally.